packages/astro/actions-module.template.mjs

```diff
@@ -0,0 +1,49 @@
+import { ActionError, callSafely } from 'astro:actions';
+
+function toActionProxy(actionCallback = {}, aggregatedPath = '/_actions/') {
+	return new Proxy(actionCallback, {
+		get(target, objKey) {
+			const path = aggregatedPath + objKey.toString();
+			if (objKey in target) {
+				return target[objKey];
+			}
+			async function action(param) {
+				if (import.meta.env.SSR) {
+					throw new ActionError({
+						code: 'BAD_REQUEST',
+						message:
+							'Action unexpectedly called on the server. If this error is unexpected, share your feedback on our RFC discussion: https://github.com/withastro/roadmap/pull/912',
+					});
+				}
+				const headers = new Headers();
+				headers.set('Accept', 'application/json');
+				let body = param;
+				if (!(body instanceof FormData)) {
```

Comment:
Should we force the `application/x-www-form-urlencoded` header when **it's a `FormData`**?

Reply:
Oh, I would actually prefer to force `multipart/form-data` to ensure `File` objects are encoded correctly. Is there a reason to prefer `urlencoded`?

Reply:
I think FormData has to be `multipart/form-data`.

Reply:
Gotcha. I know the `fetch()` function automatically applies that header when a `FormData` object is used.